### DISCUSSION TITLE
Add an optional feature to enable automatic deletion after successful backup in settings

### DISCUSSION THREAD

**Discussion by zevi7121** [immich-app/immich#14975] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Description:

I suggest adding an optional feature in the Immich settings that allows enabling automatic deletion of files from the device after they are successfully backed up.

The feature will be manual: users can enable it only if they want to.

Similar to the "Free up space" option in Google Photos, this would help users save storage on their phone automatically.

The feature will appear in the settings menu, and users can toggle it on or off as needed.

Thank you for considering this!

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#165
